manifests/server/options.pp

```diff
@@ -20,6 +20,7 @@
 define dns::server::options(
   $forwarders = [],
   $listen_on = [],
+  $listen_on_port = undef,
```

Comment:
ok, will do.  the docs for `dns::server::options` definitely need some other work too, though, as there are a bunch of options that aren't documented (`$allow_recursion`, `$check_names_master`, `$check_names_slave`, `$check_names_response`, `$allow_query`) and docs for some options that don't actually exist there (`$group`, `$owner`).